### DISCUSSION TITLE
Abnormal Temperature changes

### DISCUSSION THREAD

**Discussion by Mrzhang-hub** [pybamm-team/PyBaMM#2020] [Q&A]
Recently I have run a driving cycle lasted for about 4 hours. To my surprise, the temperature only changes a little bit, which is not reasonable. (Thermal:'lumped')
```

EM_LLI = pybamm.lithium_ion.SPMe({
        "lithium plating": "irreversible",
        "lithium plating porosity change": "true",
        "thermal": "lumped"})
parameter_values = pybamm.ParameterValues('Chen2020_plating')
```

![temperature](https://user-images.githubusercontent.com/59716473/163406457-a45144fd-ba9b-4337-9b2c-f7a8f1ec588c.png)
![current](https://user-images.githubusercontent.com/59716473/163406540-2eded763-eb8b-44a4-9706-cd62ae764bf2.png)

**Comment by brosaplanella** [comment]
That would depend on the parameters. The LG M50 cannot provide such high currents and voltages, so I assume you have changed some of the parameters. It would be good to know what's the response of a single cell and what are the thermal parameters (volume, surface area and heat transfer coefficient).

**Reply by Mrzhang-hub** [reply]
The current and voltages are acquired through 4S12P. So the current and voltages for each single cell are about  2-3 A and  3-4 V, which is in a normal level. And I have not changed any of the parameters in "Chen2020_plating" except current value.

**Reply by brosaplanella** [reply]
2-3 A is around C/2 for the LGM50, so I would not expect a lot of heating. Still, I would expect a bit more. Can you check what's the value for the `"cell geometry"` option, and what are the values for `"Cell cooling surface area [m2]"`, `"Cell volume [m3]"` and `"Total heat transfer coefficient [W.m-2.K-1]"`?

**Reply by Mrzhang-hub** [reply]
```
'Cell cooling surface area [m2]': 0.00531,
 'Cell volume [m3]': 2.42e-05,
"Total heat transfer coefficient [W.m-2.K-1]：10.0
```
As for the `Cell geometry` , I did not set this parameter in options(In my view, it is the shape of LGM50, which is  a cylinder).

**Reply by brosaplanella** [reply]
In recent versions, it should be `arbitrary` which with these parameters corresponds to a cylindrical cell, but work checking in case it is `pouch` for some reason, as this would have a different cooling.

In addition, can you send the plot for the current which is bigger (alternative you can zoom into a subset only)? From the plot above it is hard to get a grasp of how the current behaves.

**Reply by Mrzhang-hub** [reply]
![Batt_I_single cell](https://user-images.githubusercontent.com/59716473/164019346-64554dd6-9d64-4746-804f-5b87edffb2e6.png)

This is what the current of single cell looks like .

**Reply by brosaplanella** [reply]
Ok, then the results above make sense I think. Note that the I_cell is around C/5 for most of the time, and even the largest spikes do not reach 1C, so the heat generated will be really small.

**Reply by Mrzhang-hub** [reply]
I got it.  Really grateful for your help and patience！ Now I know how to revise my model and continue my work.
